server/services/visitService.ts

```diff
@@ -70,13 +70,29 @@ export default class VisitService {
     return application
   }
 
-  async bookVisit({ applicationReference }: { applicationReference: string }): Promise<VisitDto> {
+  async bookVisit({
+    applicationReference,
+    actionedBy,
+  }: {
+    applicationReference: string
+    actionedBy: string
+  }): Promise<VisitDto> {
     const token = await this.hmppsAuthClient.getSystemClientToken()
     const orchestrationApiClient = this.orchestrationApiClientFactory(token)
 
-    const visit = orchestrationApiClient.bookVisit({ applicationReference })
+    const visit = orchestrationApiClient.bookVisit({ applicationReference, actionedBy })
 
     logger.info(`Visit application '${applicationReference}' booked as visit '${(await visit).reference}'`)
     return visit
   }
+
+  async getFuturePublicVisit(bookerReference: string): Promise<VisitDto[]> {
```

Comment:
```suggestion
  async getFuturePublicVisits(bookerReference: string): Promise<VisitDto[]> {
```
Should probably pluralise the method name because it's returning an array of visits.